File: application/playmaker_v0.1/src/main/java/com/teampress/database/model/system/Parameter.java
package com.teampress.database.model.system;

import com.teampress.database.model.BaseModel;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;


/**
 * The persistent class for the PARAMETERS database table.
 * 
 */
@Entity
@Table(name="PARAMETER", schema="teampress")
@Cacheable(false)
@NoArgsConstructor
@Getter
@Setter
public class Parameter extends BaseModel {
	private static final long serialVersionUID = 1L;

	@Column(name="CODE", nullable=false, length=255)
	private String code;

	@Column(name="`GROUP`", nullable=false, length=255)
	private String group;

	@Column(name="VALUE", length=255)
	private String value;

	@Column(name="ATTR1", length=255)
	private String attr1;

	@Column(name="ATTR2", length=255)
	private String attr2;

	@Column(name="ATTR3", length=255)
	private String attr3;

	@Column(name="ATTR4", length=255)
	private String attr4;

	@Column(name="ATTR5", length=255)
	private String attr5;

	@Column
	private boolean deletable;
}

File: application/teampress/src/main/webapp/assets/js/admin/Parameter.js
$(document).ready(function () {
    $().ready(function () {
        createDataTable('table')
    });
});

function del(id){
    $.post(
        "/sys/param/del",
        {
            id: id
        },
        function (result) {
            window.location = '/sys/param';
        });
}

function get(id){
    $.post(
        "/sys/param/get",
        {
            id: id
        },
        function (result) {
            var p = JSON.parse(result);
            $('#id').val(p.id);
            $('#group').val(p.group);
            $('#code').val(p.code);
            $('#value').val(p.value);
            $('#attr1').val(p.a1);
            $('#attr2').val(p.a2);
            $('#attr3').val(p.a3);
            $('#attr4').val(p.a4);
            $('#attr5').val(p.a5);
        });
}
